src/components/application_manager/src/commands/mobile/on_button_press_notification.cc

```diff
@@ -106,20 +106,25 @@ void OnButtonPressNotification::Run() {
       continue;
     }
 
-    //Send ButtonPress notification only in HMI_FULL or HMI_LIMITED mode
+    // Send ButtonPress notification only in HMI_FULL or HMI_LIMITED mode
     if ((mobile_api::HMILevel::HMI_FULL != subscribed_app->hmi_level()) &&
         (mobile_api::HMILevel::HMI_LIMITED != subscribed_app->hmi_level())) {
-      LOG4CXX_WARN_EXT(logger_, "OnButtonPress in HMI_BACKGROUND or NONE");
+      LOG4CXX_WARN_EXT(logger_, "OnButtonPress notification is allowed only"
```

Comment:
@anosach-luxoft , this pull request in based on customer pull request https://github.com/smartdevicelink/sdl_core/pull/576